website/server/controllers/api-v3/members.js

```diff
@@ -358,7 +358,10 @@ function _getMembersForItem (type) {
       if (req.query.search) {
         // Creates a RegExp expression when querying for profile.name
         const escapedSearch = escapeRegExp(req.query.search);
-        query['profile.name'] = { $regex: new RegExp(escapedSearch, 'i') };
+        query.$or = [
+          { 'profile.name': { $regex: new RegExp(escapedSearch, 'i') } },
+          { 'auth.local.username': { $regex: req.query.search } },
```

Comment:
is there a reason why this isn't using `new RegExp(escapedSearch, 'i')` like the profile name search?

Reply:
I just copied it from the challenge username search about 10 lines above, but presumably, it should be using RegExp not sure if there was a reason.

Reply:
let's make it like the profile one, the escaping might not be necessary here but the `i` flag is useful